src/components/MyKiva/BadgeCompleted.vue

```diff
@@ -110,15 +106,15 @@ const badgeImage = computed(() => {
 });
 
 const badgeCategory = computed(() => badge.value?.challengeName ?? '');
-const tiers = computed(() => lendingAchievement.value?.tiers ?? []);
+const tiers = computed(() => badge.value?.tiers ?? []);
 
 const sortedTiers = computed(() => {
```

Comment:
The `sortedTiers` shouldn't be necessary anymore, since the new composable sorts the tiers.